### PR TITLE
DOE help files and normal effects plot

### PR DESCRIPTION
**Pull request by JTPetter**
- Added help files for DoE submodule.
- Added normal probability plot of effects.
- Fixed bug where DoE analysis would not remove aliased terms when they were squared.

### CODE REVIEW

File: R/doeAnalysis.R [hunk 1]
```diff
@@ -833,6 +842,67 @@ get_levels <- function(var, num_levels, dataset) {
...
+  # median rank order function
+  x <- tDf$tValue[order(tDf$tValue)]
+  n <- length(x)
+  i <- rank(x)
+  p <- (i - 0.3) / (n + 0.4)
+  tDf$percentile <- p[order(tDf$tValue)]
+
+  # statistical significance
+  tDf$significant <- ifelse(tDf$pValue < 0.05, "S", "N")
+  tDf$labelYPos <- pnorm(qnorm(tDf$percentile) + 0.2)
```
Comment (vandenman) [inline R/doeAnalysis.R:885]: do you have a reference for this? Stuff like `pnorm(qnorm(tDf$percentile) + 0.2)` looks a bit weird (why `+ 0.2`?). Also, you need to prefix with `stats::`, so `stats::pnorm(stats::qnorm(tDf$percentile) + 0.2)`

**Review by vandenman — CHANGES_REQUESTED** [review]
one small remark, otherwise this looks fine.

File: R/doeAnalysis.R (hunk 1 above)
Comment (JTPetter) [reply R/doeAnalysis.R]: If you mean the median rank function, yes, there are references for that (https://support.minitab.com/en-us/minitab/help-and-how-to/graphs/probability-plot/methods-and-formulas/method-of-obtaining-probability-plot-points/ ; http://engineeredsoftware.com/nasa/pe_median.htm). 

The `stats::pnorm(stats::qnorm(tDf$percentile) + 0.2)` is just some offset that seems to work reasonably well for placing the labels above the points. I found it easier/clearer to use a number on the percentage scale and then transform it than offsetting by an already transformed number. 

I have added a comment and the prefixes.

**Comment by JTPetter** [comment]
@vandenman Thanks, I have addressed your feedback and added another small fix

**Review by vandenman — APPROVED** [review]

**Pull request merged**